components/nexus-core/src/main/java/org/sonatype/nexus/proxy/item/uid/IsRemotelyAccessibleAttribute.java

```diff
@@ -22,7 +23,16 @@
 public class IsRemotelyAccessibleAttribute
     implements Attribute<Boolean>
 {
+  private static final String ATTRIBUTES_REMOTELY_ACCESSIBLE = "nexus.content.attributeAccessEnabled";
+
+  private final boolean attributes;
+
+  public IsRemotelyAccessibleAttribute() {
+    this.attributes = SystemPropertiesHelper.getBoolean(ATTRIBUTES_REMOTELY_ACCESSIBLE, false);
```

Comment:
very minor: the property, the constant, and the field names could be more consistent